.github/actions/docker-run/action.yml

```diff
@@ -65,4 +68,8 @@ runs:
           cp -r /github_workspace/* /usr/app/
           cd /usr/app/
           rm -rf tt_metal tt_eager
+          if [ ${{ inputs.install_wheel }} ]; then
```

Comment:
Did we mention in the meeting if we wanted to move the artifact downloading into the docker run action?

I do not know if that makes sense but I also do not like the dependency on the wheel presence that is implicit.

Reply:
what artifact?

Reply:
The wheel artifact. This action relies on its presence but there is no explicit dependency defined in the Github Acton jobs.